src/includes/class-alma-wc-payment-gateway.php

```diff
@@ -371,18 +401,20 @@ public function validate_payment_from_ipn( $payment_id ) {
 	/**
 	 * Render payment plan with dates.
 	 *
-	 * @param string $default_plan Plan key.
+	 * @param integer $gateway_id   Gateway id.
+	 * @param string  $default_plan Plan key.
 	 *
 	 * @return void
 	 */
-	private function render_payment_plan( $default_plan ) {
+	private function render_payment_plan( $gateway_id, $default_plan ) {
```

Comment:
refactor this function to [decrease cognitive complexity](https://sonarcloud.io/project/issues?id=alma_alma-woocommerce-gateway&open=AX6bUcQ20-qjSFvPRM4L&pullRequest=36&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL)

Reply:
please refactor this function to decrease cognitive complexity